ankisyncd/collection.py

```diff
@@ -103,3 +108,19 @@ def shutdown(self):
             del self.collections[path]
             col.close()
 
+def get_collection_wrapper(config, path, setup_new_collection = None):
+    if "collection_wrapper" in config and config["collection_wrapper"]:
+        logging.info(f"Found collection_wrapper in config, using {config['collection_wrapper']} for "
```

Comment:
Minor style nit: the current codebase supports Python 3.5, while f-strings are available only in Python 3.6 and newer.

Reply:
I will remove all the f-strings